docs/developer-docs/latest/getting-started/troubleshooting.md

```diff
@@ -103,6 +103,6 @@ Due to these two issues, it is recommended you use a proxy application such as [
 
 ## Is X feature available yet?
 
-You can see the [ProductBoard roadmap](https://portal.productboard.com/strapi) to see which feature requests are currently being worked on and which have not been started yet.
+You can see our [Public roadmap](https://feedback.strapi.io/) to see which feature requests are currently being worked on and which have not been started yet.
```

Comment:
Thanks for fixing the link, @vcoisne.
Regarding "our" vs "the", I think it would be more inclusive if we keep "the". Just like we use "the community" and not "our community". WDYT?